cluster-http/src/main/scala/akka/cluster/http/management/ClusterHttpManagement.scala

```diff
@@ -384,21 +387,41 @@ class ClusterHttpManagement(
         case (None, None) ⇒ ClusterHttpManagementRoutes(cluster)
       }
 
-      val routes = RouteResult.route2HandlerFlow(clusterHttpManagementRoutes)
+      // TODO instead of binding to hardcoded things here, discovery can also be used for this binding!
+      val hostname = settings.ClusterHttpManagementHostname
+      val port = settings.ClusterHttpManagementPort
+      // Basically: "give me the SRV host/port for the port called `akka-bootstrap`"
+      // discovery.lookup("_akka-bootstrap" + ".effective-name.default").find(myaddress)
+      // ----
+
+      // FIXME -- think about the style of how we want to make these available
+      // I was rather thinking that management extensions should be able to be registered somehow
+      // and then be included in here
+      val bootstrapSettings = ClusterBootstrapSettings(
+          ConfigFactory
+            .parseString("akka.cluster.bootstrap.contact-point.port-fallback = ")
```

Comment:
that looks strange, no value for the port-fallback?

Reply:
yeah that's from my "let me push before weekend" now cleanedup